### DISCUSSION TITLE
Branching

### DISCUSSION THREAD

**Discussion by m-clark** [m-clark/book-of-models#4] [General]
For discussion:

- Branch each chapter (or)
- Branch our separate developments (seth and michael branches)
- Use a dev branch that all other branches branch off of so that there is always a viable 'CRC' edition via main

I kind of like the idea of branching chapters, that would pretty much ensure a working product is always available, and probably silo any potential merge conflicts to one chapter.  Name branches would be simpler, but I can see either of us doing something quickly or hotfixing or forgetting to push recent changes or something that could cause conflicts (I can always find ways to muck up git). Thoughts?

**Comment by saberry** [comment]
Chapter branches should be fine! I think it has a more reasonable flow than name-based branches.
